src/routes/send-picture.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Button, StyleSheet, Image, View, ToastAndroid, ImageStore } from 'react-native';
+import { ImageManipulator } from 'expo';
+
+import colors from '../assets/colors.js'
+import metrics from '../themes/metrics.js'
+
+import Navbar from '../components/Navbar';
+
+class SendPicture extends React.Component {
+  state = {
+    image: this.props.navigation.getParam('image', 'NO-Image')
+  }
+  sendPicture = () => {
+    let resizedImage;
+    // Seth - resizing and compressing the image before sending, other wise the base64string is
+    //        too large to send and/or takes too long.
+    // TODO: Seth - make the resize options variables that can be set with environment variables.
+    // TODO: Seth - Trigger the load state when the image is sent to show a loading graphic.
+    ImageManipulator.manipulate(this.state.image.uri, [{resize: {width: 500}}], { compress: 0.5 })
+    .then((response) => {
+      resizedImage = response;
+      ImageStore.getBase64ForTag(
+        resizedImage.uri,
+        // getbase64ForTag SUCCESS callback
+        base64String => {
```

Comment:
this is a big function for inline, it should be defined outside of here for clarity

Reply:
would still be good to refactor this to be clearer, but HACKATHON.
Approving this PR